src/init.cpp

```diff
@@ -524,7 +526,7 @@ void SetupServerArgs()
     gArgs.AddArg("-dexstats", strprintf("Enable storing live dex data in DB (default: %u)", DEFAULT_DEXSTATS), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-blocktimeordering", strprintf("(Deprecated) Whether to order transactions by time, otherwise ordered by fee (default: %u)", false), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-txordering", strprintf("Whether to order transactions by entry time, fee or both randomly (0: mixed, 1: fee based, 2: entry time) (default: %u)", DEFAULT_TX_ORDERING), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
-    gArgs.AddArg("-ethstartstate", strprintf("Initialise Ethereum state trie using JSON input"), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-evmstartstate", strprintf("Initialise EVM state trie using JSON input"), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
This is arguably the one part I'm more inclined to agree, though can be argued that eth will remain more consistent in line with other references. 

However, we have set an rc. So, for better or worse, the window to discuss is past, unless you intend to alias this and consider deprecation, which IMO isn't worth it.